README.md

```diff
@@ -301,7 +244,7 @@ By default the logger is set to report `WARN` level events or higher, but will l
 
 ## Contributing
 
-If steps would be useful for different projects running the maze, add the to
+If steps would be useful for different projects using maze-runner, add them to
```

Comment:
```suggestion
If steps would be useful for different projects using maze-runner, to add them to
```

Reply:
I think the current change is correct?  As in, "if you want to add steps to maze runner, add them to `file`"

Reply:
Yep, sorry I misparsed that!